build_deb.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Pass any cmake options this way:
+
+# ./build_deb.sh -DFLEDGE_INSTALL=/some_path/Fledge
+
+mkdir build
```

Comment:
If build directory already exists then this statement gives error `mkdir: cannot create directory ‘build’: File exists`, so we can add `-p` flag to mkdir command.